graphql/resolvers.go

```diff
@@ -3088,6 +3088,28 @@ func (r *versionResolver) Patch(ctx context.Context, v *restModel.APIVersion) (*
 	return apiPatch, nil
 }
 
+func (r *versionResolver) ChildVersions(ctx context.Context, v *restModel.APIVersion) ([]*restModel.APIVersion, error) {
+	if !evergreen.IsPatchRequester(*v.Requester) {
+		return nil, nil
+	}
+	childPatchIds, err := r.sc.GetChildPatchIds(*v.Id)
+	if err != nil {
+		return nil, InternalServerError.Send(ctx, fmt.Sprintf("Couldn't find a patch with id: `%s` %s", *v.Id, err.Error()))
+	}
+	if len(childPatchIds) > 0 {
+		childVersions := []*restModel.APIVersion{}
+		for _, cp := range childPatchIds {
+			v, err := r.Query().Version(ctx, cp)
```

Comment:
Can we use `r.sc.FindVersionById(cp)` here instead. 
Just as an edge case check, Are we sure all child patches here will have a corresponding version? i.e all of the child patches are  activated..

Reply:
> Can we use r.sc.FindVersionById(cp) here instead.
discussed over slack and decided against it 

> Just as an edge case check, Are we sure all child patches here will have a corresponding version? i.e all of the child patches are activated..

done